### PR TITLE
TA-872 tests for hourly rates for new Immigration/Asylum codes

### PR DESCRIPTION
**Pull request by solomonahmed**
## What does this pull request do?

Please describe what you did.
Created new pricing tests for new IACD/IMCD matter type codes.  Also, re-factored pricing tests in pairing session with Glen Harper and Giuseppe Lobraico.

Also, slight re-org of folder structure.

## Why make these changes?

Please describe why the changes were needed.
New matter types are priced according to hourly rates so needed new feature tests.

## Checklist
- [x] Provided link to a JIRA ticket: https://dsdmoj.atlassian.net/browse/TA-872

### CODE REVIEW

**Review by your — DISMISSED** [review]
LGTM. I think there's a small conflict with `features/support/cwa_env.yml` but it's just a whiteline as far as I can see. Normally you would have to rebase this branch on top of master and solve the conflict manually. If you do an interactive rebase I'd suggest to remove the latest commit while rebasing, as the same changes would be brought in from the master branch (I've merged them this morning).

File: features/pricing/legal_help/immigration/fixed_fees/immigration_pricing_IMCC.feature [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    Given the following Matter Types are chosen:
+      | IMCC:IDOM |
+    And the following outcomes are bulkloaded:
+      | # | UFN        | CASE_START_DATE | PROFIT_COST | VAT_INDICATOR | SUBSTANTIVE_HEARING | ADJOURNED_HEARING_FEE | CMRH_ORAL | CMRH_TELEPHONE | 
+      | 1 | 010419/001 | 01/04/2019      | 100.0       | Y             | Y                   | 1                     | 1         | 1              |
+    When user confirms the submission
+    And user is on the pricing outcome details page
+    Then user should see the following outcomes:
+      | # | UFN        | Value       |
+      | 1 | 010419/001 | £ 1,515.20  |
```
Comment (your) [inline features/pricing/legal_help/immigration/fixed_fees/immigration_pricing_IMCC.feature:37]: Correct calculation:

527 * 1.2 + (237 + 161 + 166 + 90) * 1.2 = **1,417.20**

This value is returned correctly when running the test on UAT.

**Review by your — COMMENTED** [review]
I think there is one wrong calculation in one scenario, please check comment – spotted on UAT

File: features/pricing/legal_help/immigration/fixed_fees/immigration_pricing_IMCC.feature (hunk 1 above)
Comment (solomonahmed) [reply features/pricing/legal_help/immigration/fixed_fees/immigration_pricing_IMCC.feature:37]: Yes, you're right.  Good spot there.  It should be £1,417.20.

**Comment by solomonahmed** [comment]
Corrected the price error.  Re-ran this test against TST (i.e. features/pricing/legal_help/immigration/fixed_fees/immigration_pricing_IMCC.feature --tags @vat_indicator_add_pay) and all green now.

**Review by your — APPROVED** [review]

**Pull request merged**